python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -114,16 +114,27 @@ def get_schema(cls):
               do_acquire:
                 description: Perform target acquisition?
                 type: boolean
-                default: True
+                default: False
 
               do_take_sequence:
                 description: Take sequence of data on target?
                 type: boolean
-                default: True
+                default: False
 
               object_name:
                 description: SIMBAD query-able object name
                 type: string
+                default: ""
```

Comment:
I think you can require users to provide an object_name. To do that, remove the default field.

Reply:
object name not required